Configuration/StandardSequences/python/Digi_cff.py

```diff
@@ -42,3 +42,45 @@
     from FastSimulation.Configuration.DigiAliases_cff import loadDigiAliases
     loadDigiAliases(premix = False)
     from FastSimulation.Configuration.DigiAliases_cff import generalTracks,ecalPreshowerDigis,ecalDigis,hcalDigis,muonDTDigis,muonCSCDigis,muonRPCDigis
+
+#phase 2 common mods
+def _modifyDigitizerPhase2Common( theProcess ):
+    theProcess.load("CalibCalorimetry/HcalPlugins/Hcal_Conditions_forGlobalTag_cff")
+    theProcess.es_hardcode.toGet = cms.untracked.vstring(
```

Comment:
@Dr15Jones the issue with the es_hardcode module is that, by default, it isn't loaded at all. This is used to inject "fake" conditions for HCAL to handle configurations where database conditions aren't available or appropriate. Is there a preferred way to load a configuration file only for a specific Era?